### PR TITLE
Fixed funtionality to rename a svm in na_cdot_svm.

### PR DESCRIPTION
**Pull request by shanumondal**
Issue link: https://github.com/ansible/ansible/issues/39060

##### SUMMARY

Added new parameter "svm_new_name" to pass the new name to rename the
SVM.

##### ISSUE TYPE

 - Bugfix Pull Request

##### COMPONENT NAME

na_cdot_svm.py

##### ANSIBLE VERSION

```
ansible 2.5.0
  config file = /etc/ansible/ansible.cfg
  configured module search path = [u'/root/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python2.7/site-packages/ansible
  executable location = /usr/bin/ansible
  python version = 2.7.5 (default, Aug  4 2017, 00:39:18) [GCC 4.8.5 20150623 (Red Hat 4.8.5-16)]

```

##### ADDITIONAL INFORMATION

```

```

### CODE REVIEW

**Comment by shanumondal** [comment]
ready_for_review

**Comment by gundalow** [comment]
There was an issue in devel causing the docs failure. This has been fixed and I've triggered a rerun of CI for this PR

File: lib/ansible/modules/storage/netapp/na_cdot_svm.py [hunk 1]
```diff
@@ -213,7 +233,9 @@ def apply(self):
 
             elif self.state == 'present':
                 # Update properties
-                pass
+                if self.new_name is not None and not self.name == self.new_name:
```
Comment (Akasurde) [inline lib/ansible/modules/storage/netapp/na_cdot_svm.py]:
```
if self.new_name is not None and self.name != self.new_name
```

**Comment by shanumondal** [comment]
Can someone please review it?

**Review by amit0701 — CHANGES_REQUESTED** [review]
Thanks for taking a look at it. There are a few functional tests that fail.
The rename operation shouldnt be requiring other parameters, this should ideally work:
      na_cdot_svm:
        state: present
        name: infra_svm
        svm_new_name: infra_svm_new
        hostname: "{{inventory_hostname}}"
        username: "{{username}}"
        password: "{{password}}"

However, the current module complains that it requires
 root_volume, root_volume_aggregate, root_volume_security_style

**Comment by carchi8py** [comment]
na_cdot_svm.py has been replaced by na_ontap_svm.py which we recently added the ability to rename.

I believe this PR can be closed.

**Pull request closed without merging**